maquetado/books-list.html

```diff
@@ -9,6 +9,10 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
+  <div class="books-list-container">
```

Comment:
Same, this container is not needed since the `navbar` should be outside the books list